pbiviz.json

```diff
@@ -2,7 +2,7 @@
   "visual": {
     "name": "PowerBI-visuals-forcasting-TBATS",
     "displayName": "Forecasting TBATS",
-    "guid": "PBI_CV_8EDDC07B_TBATS_WORK",
+    "guid": "PBI_CV_8EDDC07B_TBATS",
```

Comment:
why this change?

Reply:
_WORK is debug guid
By mistake, I pushed into master  non final version of  visual